cdk/src/open-data-platform/data-plane/data-import/hander-factory.ts

```diff
@@ -0,0 +1,107 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import { ConnectionPool } from '@databases/pg';
+import * as AWS from 'aws-sdk';
+import { connectToDb } from '../schema/schema.handler';
+
+// These libraries don't have types, so they are imported in a different way.
+const { chain } = require('stream-chain');
+const { streamArray } = require('stream-json/streamers/StreamArray');
+const Batch = require('stream-json/utils/Batch');
+const Pick = require('stream-json/filters/Pick');
+
+const S3 = new AWS.S3();
+
+/**
+ * Builds a lambda handler that parses a GeoJSON file and executes a callback.
+ * @param s3Params - Details for how to get the GeoJSON file.
+ * @param callback - Function that operates on a single element from the file.
+ * @param numberOfRowsToWrite - Row limit, which can be used for testing.
+ * @returns
+ */
+export const geoJsonHanderFactory = (
```

Comment:
nit: Handler* instead of Hander here and in the file name

Reply:
Done.